editors/code/language-configuration.json

```diff
@@ -0,0 +1,39 @@
+{
+    "comments": {
+        "lineComment": "//",
+        "blockComment": [ "/*", "*/" ]
```

Comment:
Doc comments?

Reply:
`lineComment` and `blockComment` are mapped to the 2 included VS Code commands for toggling comments (from the command palette or keybindings). I don't think including them here serves any purpose except to assign those commands to the correct tokens for the language, so you are basically choosing which of your languages comment types should be triggered in these 2 cases.